src/AppInstallerCommonCore/Public/winget/UserSettings.h

```diff
@@ -10,11 +10,11 @@
 #include <variant>
 #include <vector>
 
+using namespace std::chrono_literals;
+using namespace std::string_view_literals;
+
 namespace AppInstaller::Settings
 {
```

Comment:
why we move usings outside for this one specifically?

Reply:
Putting the using statements inside the namespace imports them into that namespace.  So `AppInstaller::Settings` would now have definitions for `sv`, `ms`, etc.